Adição de confirmação antes do logout

MatheusRich opened the pull request:
## Mudanças Propostas

Adição de confirmação antes do logout.

## Tipo de Mudança

Que tipo de mudanças este Pull Request introduz ao Falko?
_Marque um `x` ao que se aplicar_

- [ ] Conserto de Bug
- [x] Nova Feature

## Checklist

_Marque um `x` ao que se aplicar. Se você não tiver certeza sobre algum dos tópicos, não hesite em perguntar. Estamos aqui para ajudar!_

- [x] O nome do pull request deve ser auto-explicativo e deve estar em português.
- [x] Os commits seguem a [política de commits do repositório](https://github.com/fga-gpp-mds/Falko-2017.2-BackEnd/wiki/Plano-de-Gerenciamento-de-Configura%C3%A7%C3%A3o-de-Software#41---pol%C3%ADtica-de-commits).
- [x] Não há erros de linter.
- [x] A build está passando (testes, code climate e codacy).
- [ ] O pull request está linkado à uma issue existente.

## Screenshots
![image](https://user-images.githubusercontent.com/20938712/35944580-1488dcb0-0c44-11e8-90e9-312bb06cf3b3.png)

MatheusRich committed:
Adding confirmation before logout

MartinsLucas reviewed (changes requested):
Por favor revise os testes referentes à funcionalidade. Eles devem ser modificados, pois o fluxo para realizar o logout foi alterado.

MatheusRich committed:
Improving Cypress logout test

Adds an assertion to the content of the window confirm. Plus, it adds and ID to the login button to make testing easier

MatheusRich commented:
@MartinsLucas Não havia necessidade de refatorar os testes, pois o Cypress é configurado para aceitar automaticamente os eventos de janela (veja no print abaixo).

![image](https://user-images.githubusercontent.com/20938712/36004220-af2f2ca4-0d18-11e8-8fce-9c21d83d90f7.png)

Entretanto, para melhorar o teste eu fiz uma asserção quanto ao conteúdo da janela.

MatheusRich committed:
Fixing logout unit test

MatheusRich commented:
@MartinsLucas Foi necessário consertar um teste unitário também. Agora a build está passando.
